### PR TITLE
Network animation

### PR DESCRIPTION
**Pull request by Eliottiosdevs**
This time I think I didn't change project files 😆

### CODE REVIEW

**Comment by Eliottiosdevs** [comment]
Closes #96, #95

File: ReviewMonitor/Reachability Manager/ReachabilityManager.swift [hunk 1]
```diff
@@ -13,16 +13,31 @@ class ReachabilityManager: NSObject {
...
-        offlineView.frame = CGRect(x: screenw / 2, y: 35, width: 200, height: 48)
+        offlineView.frame = CGRect(x: screenw / 2, y: -55, width: 200, height: 48)
         let window = UIApplication.shared.keyWindow
         window?.addSubview(offlineView)
+        UIView.animate(withDuration: 0.75, animations: {
+            offlineView.frame = CGRect(x: screenw / 2, y: 35, width: 200, height: 48)
+        }, completion: nil)
+        perform(#selector(removeOfflineView), with: nil, afterDelay: 5)
+        tap = UITapGestureRecognizer(target: self, action: #selector(removeOfflineView))
+        tap.numberOfTapsRequired = 1
```
Comment (RishabhTayal) [inline ReviewMonitor/Reachability Manager/ReachabilityManager.swift]: Do we need to set `numberOfTapsRequired`? By default it's 1

File: ReviewMonitor/Reachability Manager/ReachabilityManager.swift [hunk 2]
```diff
@@ -13,16 +13,31 @@ class ReachabilityManager: NSObject {
...
+        tap.numberOfTouchesRequired = 1
```
Comment (RishabhTayal) [inline ReviewMonitor/Reachability Manager/ReachabilityManager.swift]: Do we need to set `numberOfTouchesRequired`?

File: ReviewMonitor/Reachability Manager/ReachabilityManager.swift [hunk 3]
```diff
@@ -13,16 +13,31 @@ class ReachabilityManager: NSObject {
...
 
-    public class func removeOfflineView() {
-        offlineView.removeFromSuperview()
+    @objc public class func removeOfflineView() {
+        UIView.animate(withDuration: 0.75, animations: {
+            offlineView.frame = CGRect(x: screenw / 2, y: -55, width: 200, height: 48)
+        }, completion: nil)
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
+            offlineView.removeFromSuperview()
+            offlineView.removeGestureRecognizer(tap)
```
Comment (RishabhTayal) [inline ReviewMonitor/Reachability Manager/ReachabilityManager.swift]: Do we ***have*** to remove gesture recognizer? If we remove the `offlineView` from `superview`, shouldn't it remove the gesture recognizer as well?

**Comment by Eliottiosdevs** [comment]
Better?

**Comment by RishabhTayal** [comment]
Merged!

**Comment by Eliottiosdevs** [comment]
Thanks!

**Pull request merged**